pandas/tools/pivot.py

```diff
@@ -170,6 +170,9 @@ def pivot_table(data, values=None, index=None, columns=None, aggfunc='mean',
     if len(index) == 0 and len(columns) > 0:
         table = table.T
 
+    if isinstance(table, Series):
```

Comment:
Thanks @jreback. I found the root cause ([here](https://github.com/pandas-dev/pandas/blob/master/pandas/tools/pivot.py#L168)). I'll work on this and the tests tonight.

Once fixed, do you still want to keep the `isinstance(table, Series)` check or should I drop it?